plugins/subtitles.js

```diff
@@ -20,6 +21,22 @@ var srtToVtt = function(options, cb) {
   });
 };
 
+var findSubtitles = function(options) {
+  var videoPath = options.playlist[0].media.metadata.filePath;
```

Comment:
No, but `media.metadata` is ensured to always be there. And that's why I'm catching the risk for `filePath` not to be present on line 26.